benchmark_runner/common/template_operations/templates/vdbench/internal_data/vdbench_vm_template.yaml

```diff
@@ -3,16 +3,16 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ namespace }}
+---
 {%- endif %}
 {%- if odf_pvc == True %}
----
 kind: PersistentVolumeClaim
 apiVersion: v1
```

Comment:
It doesn't strictly matter, but conventionally the apiVersion comes first, followed by the kind.